### ISSUE TITLE
Add examples for each frequently used GAs

### ISSUE THREAD

**Issue by utensil** [pygae/galgebra#434]
I need a symbolic cheatsheet similar to https://observablehq.com/@enkimute/ganja-js-cheat-sheets for at least the GAs in https://github.com/enkimute/ganja.js#the-algebra-function . Particularly, I'm surprised that we don't have a simple `A*B` full expansion for any GA in our example.

**Comment by eric-wieser** [comment]
In the form of a notebook, perhaps?

**Comment by utensil** [comment]
Yes, definitely. I'll do this along with the proofs for them, mostly first examples here, then proofs there.
